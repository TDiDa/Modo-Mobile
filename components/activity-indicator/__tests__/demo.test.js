import demoTest from '../../../tests/shared/demoTest';

demoTest('activity-indicator');
