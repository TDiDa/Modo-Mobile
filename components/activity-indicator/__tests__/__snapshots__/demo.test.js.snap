// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/activity-indicator/demo/basic.md correctly 1`] = `
<div
  class="m-demo-list"
>
  <div
    class="m-activity-indicator"
  >
    <div
      class="m-activity-indicator-loading"
    >
      <i
        class="m-icon m-icon-loading"
        style="color:#1e9eff;font-size:16px"
      >
        <svg
          aria-hidden="true"
          class="m-icon-spin"
          data-icon="loading"
          fill="currentColor"
          height="1em"
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M512 1024c-69.1 0-136.2-13.5-199.3-40.2C251.7 958 197 921 150 874c-47-47-84-101.7-109.8-162.7C13.5 648.2 0 581.1 0 512c0-19.9 16.1-36 36-36s36 16.1 36 36c0 59.4 11.6 117 34.6 171.3 22.2 52.4 53.9 99.5 94.3 139.9 40.4 40.4 87.5 72.2 139.9 94.3C395 940.4 452.6 952 512 952c59.4 0 117-11.6 171.3-34.6 52.4-22.2 99.5-53.9 139.9-94.3 40.4-40.4 72.2-87.5 94.3-139.9C940.4 629 952 571.4 952 512c0-59.4-11.6-117-34.6-171.3-22.2-52.4-53.9-99.4-94.3-139.9-40.4-40.4-87.5-72.2-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.2C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3s-13.5 136.2-40.2 199.3C958 772.3 921 827 874 874c-47 47-101.8 83.9-162.7 109.7-63.1 26.8-130.2 40.3-199.3 40.3z"
          />
        </svg>
      </i>
    </div>
    <div
      class="m-activity-indicator-text"
    >
      加载中...
    </div>
  </div>
  <div
    class="m-activity-indicator m-activity-indicator-vertical"
  >
    <div
      class="m-activity-indicator-loading"
    >
      <i
        class="m-icon m-icon-loading"
        style="color:#f5222d;font-size:26px"
      >
        <svg
          aria-hidden="true"
          class="m-icon-spin"
          data-icon="loading"
          fill="currentColor"
          height="1em"
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M512 1024c-69.1 0-136.2-13.5-199.3-40.2C251.7 958 197 921 150 874c-47-47-84-101.7-109.8-162.7C13.5 648.2 0 581.1 0 512c0-19.9 16.1-36 36-36s36 16.1 36 36c0 59.4 11.6 117 34.6 171.3 22.2 52.4 53.9 99.5 94.3 139.9 40.4 40.4 87.5 72.2 139.9 94.3C395 940.4 452.6 952 512 952c59.4 0 117-11.6 171.3-34.6 52.4-22.2 99.5-53.9 139.9-94.3 40.4-40.4 72.2-87.5 94.3-139.9C940.4 629 952 571.4 952 512c0-59.4-11.6-117-34.6-171.3-22.2-52.4-53.9-99.4-94.3-139.9-40.4-40.4-87.5-72.2-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.2C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3s-13.5 136.2-40.2 199.3C958 772.3 921 827 874 874c-47 47-101.8 83.9-162.7 109.7-63.1 26.8-130.2 40.3-199.3 40.3z"
          />
        </svg>
      </i>
    </div>
    <div
      class="m-activity-indicator-text"
    >
      loading...
    </div>
  </div>
</div>
`;

exports[`renders ./components/activity-indicator/demo/carousel.md correctly 1`] = `
<div
  class="m-demo-list"
>
  <div
    class="m-activity-indicator"
  >
    <div
      class="m-activity-indicator-loading"
    >
      <div
        class="m-activity-indicator-carousel"
      >
        <svg
          class="m-activity-indicator-carousel-svg"
          fill="#1e9eff"
          style="width:64px;height:16px"
          viewBox="0 0 64 16"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="8"
            cy="8"
            r="8"
          >
            <animate
              attributeName="fill-opacity"
              attributeType="XML"
              begin="0s"
              calcMode="linear"
              dur="1s"
              repeatCount="indefinite"
              values="1;0.8;0.6;0.6;0.6;0.8;1"
            />
            <animate
              attributeName="r"
              attributeType="XML"
              begin="0s"
              calcMode="linear"
              dur="1s"
              repeatCount="indefinite"
              values="8;6.4;4.8;4.8;4.8;6.4;8"
            />
          </circle>
          <circle
            cx="32"
            cy="8"
            r="8"
          >
            <animate
              attributeName="fill-opacity"
              attributeType="XML"
              begin="0s"
              calcMode="linear"
              dur="1s"
              repeatCount="indefinite"
              values="0.6;0.8;1;0.8;0.6;0.6;0.6"
            />
            <animate
              attributeName="r"
              attributeType="XML"
              begin="0s"
              calcMode="linear"
              dur="1s"
              repeatCount="indefinite"
              values="4.8;6.4;8;6.4;4.8;4.8;4.8"
            />
          </circle>
          <circle
            cx="56"
            cy="8"
            r="8"
          >
            <animate
              attributeName="fill-opacity"
              attributeType="XML"
              begin="0s"
              calcMode="linear"
              dur="1s"
              repeatCount="indefinite"
              values="0.6;0.6;0.6;0.8;1;0.8;0.6"
            />
            <animate
              attributeName="r"
              attributeType="XML"
              begin="0s"
              calcMode="linear"
              dur="1s"
              repeatCount="indefinite"
              values="4.8;4.8;4.8;6.4;8;6.4;4.8"
            />
          </circle>
        </svg>
      </div>
    </div>
    <div
      class="m-activity-indicator-text"
    />
  </div>
  <button
    class="m-button m-button-primary"
    type="button"
  >
    <div
      class="m-button-inner"
    >
      <span>
        确认支付
      </span>
    </div>
  </button>
</div>
`;
